packages/api/src/error-handler.js

```diff
@@ -8,7 +8,7 @@ import { HTTPError, PinningServiceApiError } from './errors.js'
  * @param {Request} request
  */
 export function errorHandler (err, { log }, request) {
-  console.error(err.stack)
+  console.error(err)
```

Comment:
this make it so you can actually see the error.cause in the console log when debugging. it's very helpful.